modules/cluster-logging-collector-log-forward-logs-from-application-pods.adoc

```diff
@@ -0,0 +1,63 @@
+[id="cluster-logging-collector-log-forward-logs-from-application-pods_{context}"]
+= Forwarding application logs from specific pods
+
+As a cluster administrator, you can use Kubernetes pod labels to gather log data from specific pods and forward it to a log collector.
+
+Suppose that you have an application composed of pods running alongside other pods in various namespaces. If those pods have labels that identify the application, you can gather their log data and output it to a specific log collector.
```

Comment:
Suggest avoiding usage of 'Suppose that' - conversational tone. Pg 35 ISG: Although a friendly or conversational style might be appropriate in some contexts, do not write content that replicates the way that people sometimes speak. On the page or screen, this type of writing can seem rambling or verbose.

This entire paragraph repeats what you say in the first sentence very crisply.
Can we remove this sentence and retain only the extra information in this section? Something on the lines of: This is particularly useful when you have an application composed of pods running alongside other pods in various namespaces. 
It also makes the "why" part clear to the user.

Reply:
I understand your reasoning, I leave it up to you. Just quickly looked up the OpenShift docs though, I don't see it used much.